mcc/resources/schemas/dbscripts/postgresql/mcc-20.003-20.004.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE mcc.animalRequests (
+	rowid 									SERIAL 				NOT NULL,
+	objectid 								VARCHAR(40)   NOT NULL,
+	lastName 								VARCHAR(1000) NOT NULL,
+	firstName 							VARCHAR(1000) NOT NULL,
+	middleInitial 					VARCHAR(10) 	NOT NULL,
+	isPrincipalInvestigator BOOLEAN 			NOT NULL,
+	institutionName 				VARCHAR(1000) NOT NULL,
+	institutionCity 				VARCHAR(1000) NOT NULL,
+	institutionState 				VARCHAR(1000) NOT NULL,
+	institutionCountry 			VARCHAR(1000) NOT NULL,
+	institutionType 				VARCHAR(50) 	NOT NULL,
+	officialLastName 				VARCHAR(1000) NOT NULL,
+	officialFirstName 			VARCHAR(1000) NOT NULL,
+	officialEmail 					VARCHAR(1000) NOT NULL,
+	coInvestigators 				TEXT,
+	fundingSource 					VARCHAR(50) 	NOT NULL,
+	experimentalRationale 	VARCHAR(4000) NOT NULL,
+	numberOfAnimals 				SMALLINT 			NOT NULL,
+	otherCharacteristics 		VARCHAR(4000) NOT NULL,
+	methodsProposed 				VARCHAR(4000) NOT NULL,
+	collaborations 					VARCHAR(4000) NOT NULL,
+	isBreedingAnimals 			BOOLEAN 			NOT NULL,
+	ofInterestCenters 			VARCHAR(4000) NOT NULL,
+	researchArea 						VARCHAR(50) 	NOT NULL,
+	otherJustification 			VARCHAR(255),
+	existingMarmosetColony  BOOLEAN 			NOT NULL,
+	existingNHPFacilities   BOOLEAN 			NOT NULL,
+	animalWellfare 					VARCHAR(4000) NOT NULL,
+	vetLastName 						VARCHAR(1000) NOT NULL,
+	vetFirstName            VARCHAR(1000) NOT NULL,
+	vetEmail 								VARCHAR(1000) NOT NULL,
+	iacucApproval 					VARCHAR(50)   NOT NULL,
+
+	CONSTRAINT PK_animalRequests PRIMARY KEY (rowid)
+);
+
+
+CREATE TABLE mcc.coinvestigators (
+	rowid 									SERIAL 				NOT NULL,
+	requestid 							VARCHAR(40)   NOT NULL,
+	lastName 								VARCHAR(1000) NOT NULL,
+	firstName 							VARCHAR(1000) NOT NULL,
+	middleInitial 					VARCHAR(10) 	NOT NULL,
+	institutionName 				VARCHAR(1000) NOT NULL,
```

Comment:
you should add these columns:

container ENTYTYID,
created datetime,
createdby int,
modified datetime,
modifiedby int

Reply:
you can add this XML block per table too:

```
            <column columnName="container">
                <isHidden>true</isHidden>
            </column>
            <column columnName="createdby">
                <isHidden>true</isHidden>
            </column>
            <column columnName="created">
                <isUserEditable>false</isUserEditable>
                <shownInInsertView>false</shownInInsertView>
                <shownInUpdateView>false</shownInUpdateView>
                <nullable>true</nullable>
                <scale>29</scale>
                <isHidden>true</isHidden>
            </column>
            <column columnName="modifiedby">
                <isHidden>true</isHidden>
            </column>
            <column columnName="modified">
                <isUserEditable>false</isUserEditable>
                <shownInInsertView>false</shownInInsertView>
                <shownInUpdateView>false</shownInUpdateView>
                <nullable>true</nullable>
                <scale>29</scale>
                <isHidden>true</isHidden>
            </column>

```